executor/executable/controllabletask.go

```diff
@@ -387,17 +392,35 @@ func (t *ControllableTask) Kill() error {
 			"targetList": cmd.TargetList,
 		}).
 		Debug("state DONE not reached, about to commit transition")
+		
+		// Call cmd.Commit() asynchronous
+		commitDone := make(chan *CommitResponse)
+    	go func() {
```

Comment:
There seems to be an indentation issue here, not sure if it's tabs or bad IDE formatting. We should always use spaces.